frontend/src/lib/components/CommandPalette/commandStyledComponents.ts

```diff
@@ -78,12 +77,6 @@ export const ResultBox = styled.div<ResultBoxProps>`
             width: 0.375rem;
         }
         `}
-    ${({ isHint }) =>
```

Comment:
We definitely need to have hints to increase commands discoverability. This will be particularly important for commands that you don't initially expect (like share feedback). Besides we already have this code

Reply:
Now we have randomized context-dependent suggestions, which are actionable as opposed to hints. An improvement would be making these suggestions smarter.